frontend/catalyst/api_extensions/callbacks.py

```diff
@@ -39,6 +41,63 @@
 
 
 ## API ##
+def accelerate(func=None, dev=None):
```

Comment:
The common pattern for our decorators explicitly separates out the optional parameters as keyword-only arguments:
```suggestion
def accelerate(func=None, *, dev=None):
```